### DISCUSSION TITLE
How to deploy tRPC client and server on different places.

### DISCUSSION THREAD

**Discussion by AxeemHaider** [trpc/trpc#3523] [Q&A] [answered]
I see tRPC is something like monorepo both server and client should have in same place. Because you have to **import types from server** is there any way I can keep my client in different place ??? Then how can I import the types from server ???

**Comment by psnehanshu** [comment, marked as answer]
You seem confused between the words "deploy" and "monorepo". Monorepo means the code for multiple projects resides in the same repo. Where you deploy them isn't the concert here. For example, you can have an express server and a react application in the repository under directories (for example) `backend` and `frontend`. But you can deploy them to different places like EC2 and S3.

If your concern is how to use tRPC when the code for the backend and frontend are on different repositories, then you're out of luck. The website [says](https://trpc.io/docs/further-reading#who-is-this-for):

> It's designed for monorepos, as you need to export/import the type definitions from/to your server.

So if you want the code to be on different repositories, you can't use tRPC.

**Comment by IRediTOTO** [comment]
Is there any way export types of trpc then we can copy it to client folder?
